Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -5914,8 +5916,8 @@ M  END
 )CTAB"_ctab;
   REQUIRE(m);
 
-  // Currently, MaeMolSupplier does not recognize "either" double bonds, this just
```

Comment:
Unrelated whitespace fixes (which made tha PR not apply cleanly for 2023_09_02 for me)

Reply:
yeah, we do these automatically as files are touched. It's an unfortunate consequence of what is otherwise a good thing (automatic code reformattiing)